boefjes/boefjes/job_handler.py

```diff
@@ -175,6 +176,8 @@ def handle(self, boefje_meta: BoefjeMeta) -> None:
             else:
                 logger.info("No results for boefje %s[%s]", boefje_meta.boefje.id, boefje_meta.id)
 
+            gc.collect()
```

Comment:
I feel deleting the known large items before running gc.collect() would be useful.

Reply:
excellent, that makes more sense!